File: natureremocontroller/src/main/java/com/fishwaffle/natureremo/controller/models/ApplianceModelAndParam.java
/*
 * Copyright (c) 2018 FishWaffle.
 */

package com.fishwaffle.natureremo.controller.models;

import java.io.Serializable;

public class ApplianceModelAndParam  implements Serializable {
    /**プリセット情報*/
    public ApplianceModel model;
    /**エアコンの設定状態*/
    public AirConParams params;
}

File: natureremocontroller/src/main/java/com/fishwaffle/natureremo/controller/models/ApplianceModelAndParam.kt
/*
 * Copyright (c) 2019 FishWaffle.
 */

package com.fishwaffle.natureremo.controller.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fishwaffle.natureremo.controller.isIgnoreUnknown
import java.io.Serializable

/**
 * @param model プリセット情報
 * @param params エアコンの設定状態
 */
@JsonIgnoreProperties(ignoreUnknown = isIgnoreUnknown)
data class ApplianceModelAndParam(var model: ApplianceModel?, var params: AirConParams?) : Serializable